CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ set(RADIUM_PLUGINS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/Plugins")
 set(RADIUM_PLUGIN_OUTPUT_PATH "${RADIUM_BUNDLE_DIRECTORY}/${CMAKE_BUILD_TYPE}/bin/Plugins")
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${RADIUM_BUNDLE_DIRECTORY}/${CMAKE_BUILD_TYPE}/bin)
 set(EXECUTABLE_OUTPUT_PATH         ${RADIUM_BUNDLE_DIRECTORY}/${CMAKE_BUILD_TYPE}/bin)
+set(RADIUM_BINARY_OUTPUT_PATH "${RADIUM_BUNDLE_DIRECTORY}/${CMAKE_BUILD_TYPE}/bin")
```

Comment:
is that different from EXECUTABLE_OUTPUT_PATH  ?

Reply:
no more than EXECUTABLE_OUTPUT_PATH is different from CMAKE_RUNTIME_OUTPUT_DIRECTORY. I'll use the latest.